plugins/in_elasticsearch/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+if(NOT FLB_METRICS)
+  message(FATAL_ERROR "Elasticsearch input plugin requires FLB_HTTP_SERVER=On.")
+endif()
+
+set(src
+  in_elasticsearch.c
+  in_elasticsearch_config.c
+  in_elasticsearch_bulk_conn.c
+  in_elasticsearch_bulk_prot.c
+  )
+
+FLB_PLUGIN(in_elasticsearch "${src}" "monkey-core-static")
```

Comment:
don't need to add `monkey-core-static`

Reply:
Removed in https://github.com/fluent/fluent-bit/pull/6685/commits/89b749ec9a2fb6abf7a2ce533a78d0ac3d33f67e.